src/workers/parser/getScopes.js

```diff
@@ -1,5 +1,3 @@
-// @flow
```

Comment:
I removed this line because we're ignoring `.*node_modules/devtools-map-bindings/src/parser.js`; otherwise, we would get this error:
<img width="759" alt="32411668-8bb97f6e-c21c-11e7-9c02-819dc0dee7fb" src="https://user-images.githubusercontent.com/4562118/32550150-0124a6b4-c4c7-11e7-96b1-354d43834fe3.png">